base/range.jl

```diff
@@ -786,3 +786,5 @@ end
 
 in{T<:Integer}(x, r::Range{T}) = isinteger(x) && !isempty(r) && x>=minimum(r) && x<=maximum(r) && (mod(convert(T,x),step(r))-mod(first(r),step(r)) == 0)
 in(x::Char, r::Range{Char}) = !isempty(r) && x >= minimum(r) && x <= maximum(r) && (mod(Int(x) - Int(first(r)), step(r)) == 0)
+
+unique(x::Range) = x
```

Comment:
Maybe this should return an `Array` instead (i.e. call `collect`)? The documentation for `unique` isn't very clear about what kind of "array" it returns. Not sure.